browser/extensions/api/brave_rewards_api.cc

```diff
@@ -608,6 +608,9 @@ ExtensionFunction::ResponseAction BraveRewardsSaveAdsSettingFunction::Run() {
       const auto is_enabled =
           params->value == "true" && ads_service_->IsSupportedLocale();
       ads_service_->SetEnabled(is_enabled);
+    } else if (params->key == "shouldShowPublisherAdsOnParticipatingSites") {
```

Comment:
Can you use `kShouldShowPublisherAdsOnParticipatingSites`?

Reply:
Fixed